src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -852,4 +853,15 @@ public function testMultipleLocationsWithSubDirectories()
         $this->assertIterator($expected, $finder);
         $this->assertIteratorInForeach($expected, $finder);
     }
+
+    public function testShellCommand()
+    {
+        $cmd = Command::create()->add('--force');
+        $this->assertSame('--force', $cmd->join());
```

Comment:
How about introducing separate `Shell\CommandTest.php` suite to cover `Command` class then? I'd be happy to work on it as it's one of the least covered class in a whole component.